library/src/main/java/com/northconcepts/templatemaster/form/CrudResource.java

```diff
@@ -47,15 +47,17 @@ public CrudResource(String singularTitle, String pluralTitle, String subUrl, For
         this.formDef = formDef;
     }
 
-    protected abstract Page<ENTITY> getPage(String keyword, String sortField, int pageNumber);
+    protected abstract Page<ENTITY> getPage(String keyword, String sortField, int pageNumber, int pageSize);
+    
+    protected abstract int getUserListPageSize(Integer pageSize);
```

Comment:
What's the purpose of the `pageSize` argument passed to this method?

Reply:
We have refactored the code.